personal_assistant/personal_assistant/urls.py

```diff
@@ -19,6 +19,7 @@
 
 urlpatterns = [
     path('', include('pa_core.urls')),
+    path('', include('contacts.urls')), #Додав файл urls зі свого застосунку contacts
```

Comment:
Пропоную префікс шляхів зазначити саме тут, щоб скоротити текст у згаданому файлі. Також коментар не вписується в обмеження по довжині рядка тож раджу його перенести вище.
```suggestion
    # Додав файл urls зі застосунку контактів.
    path('contacts/', include('pa_contact.urls')),
```